CHANGELOG.md

```diff
@@ -1,99 +1,105 @@
+# 2.1.1
+
+> Jun 04, 2021
+
+- :nut_and_bolt: **New** Added Switch.
+
 # 2.1.0
 
 > Apr 12, 2021
 
-* :tada: **Enhancement** Reintroduce the fhi-app and fhi-main-menu blocks into the package
-* :tada: **Enhancement** Remove ngDatepicker variables
-* :boom: **Breaking** Remove _fhi-app.block.scss and assosiated variables
-* :nut_and_bolt: **New** Establish CI build that runs on every push and pull-request.
-* :tada: **Enhancement** Rename Release workflow. Remove unused publish workflow.
+- :tada: **Enhancement** Reintroduce the fhi-app and fhi-main-menu blocks into the package
+- :tada: **Enhancement** Remove ngDatepicker variables
+- :boom: **Breaking** Remove \_fhi-app.block.scss and assosiated variables
```

Comment:
Could you do me an favor and remove this line even thougt it's an old error; 2.1.0 is not breaking since "**Enhancement** Reintroduce the fhi-app and fhi-main-menu blocks into the package" reverts **Breaking** Remove _fhi-app.block.scss and assosiated variables